torch/_inductor/compile_fx.py

```diff
@@ -846,7 +849,7 @@ def compile_fx_aot(
     example_inputs_: List[torch.Tensor],
     inner_compile: Callable[..., Any] = compile_fx_inner,
     config_patches: Optional[Dict[str, Any]] = None,
-):
+) -> str:
```

Comment:
sorry, with https://github.com/pytorch/pytorch/pull/107468, this is still a callable.